analytic_base_department/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2011-2018 Camptocamp SA
+# Copyright 2013 Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Base Analytic Department Categorization",
+    "version": "12.0.1.0.0",
+    "author": "Camptocamp, Daniel Reis, Odoo Community Association (OCA)",
```

Comment:
@anandkansagra `Daniel Reis`  name in an author.
@dreispt can you please send company name?

Reply:
There is no company name to go there.